packages/azure-http-specs/spec-summary.md

```diff
@@ -916,6 +916,153 @@ Expected response body:
 }
 ```
 
+### Azure_ResourceManager_Resources_LocationResources_createOrUpdate
+
+- Endpoint: `put https://management.azure.com`
+
+Resource PUT operation.
+Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource
+Expected query parameter: api-version=2022-12-01-preview
+Expected request body:
+
+```json
+{
+  "properties": {
+    "description": "valid"
+  }
+}
+```
+
+Expected response body:
+
+```json
+{
+  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource",
+  "name": "resource",
+  "type": "Azure.ResourceManager.Resources/locationResources",
+  "properties": {
+    "description": "valid",
+    "provisioningState": "Succeeded",
+  },
+  "systemData": {
+    "createdBy": "AzureSDK",
+    "createdByType": "User",
+    "createdAt": <any date>,
+    "lastModifiedBy": "AzureSDK",
+    "lastModifiedAt": <any date>,
+    "lastModifiedByType": "User",
+  }
+}
+```
+
+### Azure_ResourceManager_Resources_LocationResources_delete
+
+- Endpoint: `delete https://management.azure.com`
+
+Resource DELETE operation.
+Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource
+Expected query parameter: api-version=2023-12-01-preview
+Expected response status code: 204
+
+### Azure_ResourceManager_Resources_LocationResources_get
+
+- Endpoint: `get https://management.azure.com`
+  Resource GET operation.
+  Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource
+  Expected query parameter: api-version=2023-12-01-preview
+  Expected response body:
+
+```json
+{
+  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource",
+  "name": "resource",
+  "type": "Azure.ResourceManager.Resources/locationResources",
+  "properties":{
+    "description": "valid",
+    "provisioningState": "Succeeded"
+  },
+  "systemData": {
+    "createdBy": "AzureSDK",
+    "createdByType": "User",
+    "createdAt": <any date>,
+    "lastModifiedBy": "AzureSDK",
+    "lastModifiedAt": <any date>,
+    "lastModifiedByType": "User",
+  }
+}
+```
+
+### Azure_ResourceManager_Resources_LocationResources_listByParent
```

Comment:
run `pnpm regen-docs`

Reply:
I forgot to modify the summary. Now fixed it by [commit#9ec50e4](https://github.com/Azure/typespec-azure/pull/1932/commits/9ec50e4a8b8a49c628cb3e21cb5b93acfdede548)